external/anomaly/tests/helpers/train.py

```diff
@@ -22,10 +22,7 @@
 import time
```

Comment:
Not sure if we still need this file. `sample.py` is for the same purpose and more up-to-date. It might be an idea to deprecate this